codecept

```diff
@@ -25,4 +25,5 @@ $app->add(new Codeception\Command\GenerateGroup('generate:groupobject'));
 $app->add(new Codeception\Command\GeneratePageObject('generate:pageobject'));
 $app->add(new Codeception\Command\GenerateStepObject('generate:stepobject'));
 $app->add(new Codeception\Command\GenerateEnvironment('generate:environment'));
+$app->registerCustomCommands();
```

Comment:
Probably you will need to parse args manually, the same thing I'm doing in Robo. Let me show you sample code: https://github.com/codegyre/robo/blob/master/src%2FRunner.php#L108